glean-core/android/src/main/java/mozilla/telemetry/glean/private/TimeUnit.kt

```diff
@@ -9,11 +9,32 @@ package mozilla.telemetry.glean.private
  * the Timespan and TimingDistribution metric types.
  */
 enum class TimeUnit {
+    /**
+     * Truncate to nanosecond precision.
```

Comment:
This is not strictly true anymore for `TimingDistribution`. I'd leave "truncate" out, and use the wording "Represents a xx precision"